readme.md

```diff
@@ -54,8 +54,9 @@ If you need better performance you should check out [NHunspell](https://www.nuge
 
 | Benchmark | [WeCantSpell.Hunspell](https://www.nuget.org/packages/WeCantSpell.Hunspell/) net6 | [WeCantSpell.Hunspell](https://www.nuget.org/packages/WeCantSpell.Hunspell/) net48 | [NHunspell](https://www.nuget.org/packages/NHunspell/) |
 |---------------------|--------------|--------------|--------------|
```

Comment:
Some documentation on the new options overload and CancellationToken behavior could be added to this file.

Reply:
I think it should be intuitive enough on it's own now that CancellationToken is it's own parameter, separate from options.